release.html

```diff
@@ -0,0 +1,23 @@
+---
+layout: release
+title: Releases
+miniBanner: true
+---
+<h1> OSMT Releases</h1>
+<p class="osmtContent"><em>OSMT Release provided new things added to the OSMT open-source community.</em></p>
+<h2> OSMT 1.0.5 has been released</h2>
+<p><em>RELEASES | John Kallies | OCTOBER 25, 2021</em> </p>
+<p>On behalf of the community, I am pleased to announce that the 1.0.5 release of <a href="https://github.com/wgu-opensource/osmt/releases" target="_blank"> OSMT </a> is available today. The release can be found in <a href="https://github.com/wgu-opensource/osmt/releases" target="_blank">Github</a>. You can check out the 1.0.5 <a href="https://github.com/wgu-opensource/osmt/releases" target="_blank"> release notes </a>for more information.</p>
```

Comment:
1. OSMT link should point to : https://github.com/wgu-opensource/osmt/tree/master
2. Need to re-phase "The release can be found in <a href="https://github.com/wgu-opensource/osmt/releases" target="_blank">Github</a>" to point to dockerhub image.